libctru/Doxyfile

```diff
@@ -1983,15 +1983,15 @@ ENABLE_PREPROCESSING   = YES
 # The default value is: NO.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-MACRO_EXPANSION        = NO
+MACRO_EXPANSION        = YES
```

Comment:
Looks like `Doxyfile` got inadvertently changed during one your commits. Could you separate this into a different PR?